extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -194,7 +196,25 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 							continue;
 						}
 						try {
+							// Prompt to include password in connection string
+							if (connectionInfo.authenticationType !== 'SqlLogin') {
+								connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, true, false);
```

Comment:
includePassword should be false (it shouldn't matter but better to be safe here)